src/storage/src/hummock/local_version.rs

```diff
@@ -223,18 +231,24 @@ impl LocalVersion {
             swap(&mut shared_buffer_to_sync, &mut self.shared_buffer);
             let insert_result = self.sync_uncommitted_data.insert(
                 new_epoch,
-                SyncUncommittedData::new(new_epoch, shared_buffer_to_sync),
+                SyncUncommittedData::new(new_epoch, last_epoch, shared_buffer_to_sync),
             );
             assert_matches!(insert_result, None);
             self.max_sync_epoch = new_epoch;
```

Comment:
I think `max_sync_epoch` and `sealed_epoch` are the same after this PR. Let's keep `sealed_epoch` only.

Reply:
`max_sync_epoch` and `sealed_epoch` are the same only when the `is_checkpoint` of `seal_epoch` call is always true. In the future when we support barrier and checkpoint decouple, they are different. I would suggest keep them both now.

Reply:
I see. Makes sense.